scripts/python3.8/linux/build.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# This scripts builds a Cura AppImage using a docker container. The docker base
+# image it uses is "cura-build-env:centos7", which can be created with the
+# cura-build-environment repository. The AppImage will be created under the
+# directory "appimages", which will be created if it's not present. Note that
+# the AppImage being created will not be signed.
+#
+
+set -e
+
+# Get where this script resides
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../../.."
+
+# Cura release configurations
+CURA_BUILD_ENV_DOCKER_IMAGE="${CURA_BUILD_ENV_DOCKER_IMAGE:-ultimaker/cura-build-environment:centos7-python3.8-qt5.15.2}"
+
+CURA_BRANCH_OR_TAG="${CURA_BRANCH_OR_TAG:-master}"
+URANIUM_BRANCH_OR_TAG="${URANIUM_BRANCH_OR_TAG:-master}"
+CURAENGINE_BRANCH_OR_TAG="${CURAENGINE_BRANCH_OR_TAG:-master}"
+CURABINARYDATA_BRANCH_OR_TAG="${CURABINARYDATA_BRANCH_OR_TAG:-master}"
+FDMMATERIALS_DATA_BRANCH_OR_TAG="${FDMMATERIALS_BRANCH_OR_TAG:-master}"
+LIBCHARON_BRANCH_OR_TAG="${LIBCHARON_BRANCH_OR_TAG:-master}"
+
+CURA_VERSION_MAJOR="${CURA_VERSION_MAJOR:-9}"
+CURA_VERSION_MINOR="${CURA_VERSION_MINOR:-9}"
+CURA_VERSION_PATCH="${CURA_VERSION_PATCH:-9}"
+CURA_VERSION_EXTRA="${CURA_VERSION_EXTRA:-Python3.8-Qt5.15.2}"
+
+CURA_BUILD_TYPE="${CURA_BUILD_TYPE}"
+CURA_NO_INSTALL_PLUGINS="${CURA_NO_INSTALL_PLUGINS}"
+
+CURA_CLOUD_API_ROOT="${CURA_CLOUD_API_ROOT:-https://api.ultimaker.com}"
+CURA_CLOUD_API_VERSION="${CURA_CLOUD_API_VERSION:-1}"
+CURA_CLOUD_ACCOUNT_API_ROOT="${CURA_CLOUD_ACCOUNT_API_ROOT:-https://account.ultimaker.com}"
+CURA_MARKETPLACE_ROOT="${CURA_MARKETPLACE_ROOT:-https://marketplace.ultimaker.com}"
+CURA_DIGITAL_FACTORY_URL="${CURA_DIGITAL_FACTORY_URL:-https://digitalfactory.ultimaker.com}"
+
+CURA_ENABLE_DEBUG_MODE="${CURA_ENABLE_DEBUG_MODE:-ON}"
+CURA_ENABLE_CURAENGINE_EXTRA_OPTIMIZATION_FLAGS="${CURA_ENABLE_CURAENGINE_EXTRA_OPTIMIZATION_FLAGS:-ON}"
```

Comment:
As far as I know this never needs to be off any more.

Reply:
Sure, I am also fine with that.